terraform/development/shared/main.tf

```diff
@@ -187,15 +188,51 @@ module "front_door" {
   initial_create = var.initial_create
 }
 
-module "monitoring" {
-  source = "../../modules/monitoring"
+module "monitoring_secrets" {
+  source = "../../modules/monitoring_secrets"
 
-  create_email_subscription = false
+  prefix = local.prefix
+
+  initial_create = var.initial_create
+}
+
+module "monitoring_topic_main" {
+  source = "../../modules/monitoring_topic"
+
+  create_email_subscription = true

```

Comment:
We actually do have a subscription already, but it's not managed in terraform. I think it should be brought in.